mappings/net/minecraft/screen/ScreenHandlerType.mapping

```diff
@@ -12,6 +24,10 @@ CLASS net/minecraft/class_3917 net/minecraft/screen/ScreenHandlerType
 		ARG 0 syncId
 		ARG 1 playerInventory
 	CLASS class_3918 Factory
+		COMMENT A functional interface that creates a screen handler instance on the client.
+		COMMENT
+		COMMENT <p>Screen handlers usually have a constructor that can be used as an implementation.
+		COMMENT See the note at {@link ScreenHandler}.
```

Comment:
```suggestion
		COMMENT See the note on {@link ScreenHandler}.
```
"at" sounds a bit weird to me? Not sure if "on" is entirely correct either though 😅